lib/components/CTACard/styles.ts

```diff
@@ -0,0 +1,30 @@
+import { makeStyles } from '@apisuite/fe-base'
+
+export default makeStyles((theme) => ({
+  ctaCardButton: {
+    color: `${theme.palette.secondary.contrastText} !important`,
+    textDecoration: 'none',
+  },
+
+  ctaCardContentsContainer: {
```

Comment:
can't you use [Card](https://material-ui.com/components/cards) from `@apisuite/fe-base` instead of recreating it in css? Because we have shadows configuration and this one has an hardcoded one.